lib/clock.rb

```diff
@@ -0,0 +1,16 @@
+require "clockwork"
+require "delayed_job"
+require "delayed_job_active_record"
+
+require_relative "../app"
+require_relative "../app/jobs/fetch_feed_job"
+require_relative "../app/repositories/feed_repository"
+
+include Clockwork
+
+fetch_interval = (ENV["FETCH_INTERVAL"] || 60).to_i
```

Comment:
OK, I've used the VPS strategy and used 10 min as default fetching frequency. Does it sounds good?